docs/book/v3/features/error-handling.md

```diff
@@ -154,6 +156,9 @@ allowing the listener the ability to introspect the generated response as well.
 As an example, you could create a logging listener as follows:
 
 ```php
+<?php
+declare(strict_types=1);
+
```

Comment:
Excerpt from the Writers Guide:

> To standardize the examples and reduce unnecessary repetition, the following rules must be observed:
> 
> - PHP's open and close tags are only allowed in code examples for view scripts
> - the declare construct must be omitted

Reply:
Might it be worth adding a documented exception in this case? As always, I'm happy to go with the group consensus.

Reply:
On reflection, you're right. Though, I'm keen to evolve the Writer's Guide.